examples/astro/src/utils/client.ts

```diff
@@ -47,10 +47,10 @@ const fetchPageData = async (params: any) => {
     return { pageAsset };
   } catch (error: any) {
     if (error?.status === 404) {
-      return { pageAsset: null, error: null };
+      return { pageAsset: undefined, error: null };
```

Comment:
this should be null, null

Reply:
We had a type error because this was null, let me see what I can do

Reply:
I think is better to use `undefined`, I organized the typing better